manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool

```

Comment:
`(PCST)` was removed from the PackageIdentifier. Please update file name and path to match.